indico/modules/events/registration/util.py

```diff
@@ -438,8 +439,12 @@ def generate_ticket_qr_code(registration):
         "event_id": unicode(registration.event_new.id),
         "server_url": Config.getInstance().getBaseURL()
     }
-    json_qr_data = json.dumps(qr_data)
-    qr.add_data(json_qr_data)
+    extended_qr = signals.event.designer.generate_ticket_qr_code.send(registration, qr_data=qr_data)
+    if extended_qr:
+        qr = extended_qr[0][1]
```

Comment:
does it really make sense to override the generation of the QR code here instead of just generating it with the updated data dict?